docs/content/users/quickstart.md

```diff
@@ -397,6 +397,33 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
     ```
 
+!!!tip "Add Vite support?"
+    Since Laravel v9.19.0, Vite will be shipped as default [asset bundler](https://laravel.com/docs/11.x/vite). There are two tweaks needed for usage in DDEV.
+
+    First, expose the port in `.ddev/config.yaml` and run `ddev restart`:
+
+    ```yaml
+    web_extra_exposed_ports:
+      - name: vite
+        container_port: 5173
+        http_port: 5172
+        https_port: 5173
+    ```
+    
+    Secondly, add this to your `vite.config.js` and start the Vite devserver with `ddev npm run dev`:
+
+    ```js
+    server: {
+        host: '0.0.0.0',
+        strictPort: true,
+        port: 5173,
+        hmr: {
+            protocol: 'wss',
+            host: `${process.env.DDEV_HOSTNAME}`
+        }
+    }
```

Comment:
Curious. Your blog post say:

```
    server: {
        // respond to all network requests:
        host: '0.0.0.0',
        port: port,
        strictPort: true,
        // Defines the origin of the generated asset URLs during development
        origin: origin
    },
```

This PR uses a similar method to https://github.com/torenware/ddev-viteserve. They both appear to work. Does anyone know the difference?

Reply:
Hi! :) 

Laravel has a special Vite plugin (npm), which creates a `public/hot` file with the devserver URL and port in it. My current knowledge is that the special configuration via `.hmr` is necessary for Laravel, but I haven't invested much time lately to this. 

Btw: Viteserve seems not be maintained anymore + I personally found it to be a drawback to not see the logs / errors (see issues).

Reply:
I also have a special section for Laravel in my blogpost (bit below the general instructions)

Reply:
@tyler36 Is this interesting for you to know if the general approach works as well in the meantime? (Laravel updated their vite plugin as well) Could invest some time to test it today.

Reply:
Okay, first quick try with the general config (as also used in viteserve and in my general blog post tutorial) - seems to work now, guess they updated the laravel vite plugin 🎉 Awesome, no special docs for Laravel anymore.  🎉

Thanks for commenting!

Reply:
I have an alpha addon that uses the blog post as a reference guide hence the curiosity.
Thank you for the updates!